app/controllers/application_controller.rb

```diff
@@ -105,7 +105,7 @@ def flash_to_headers
   private
 
   def reset_tenant_if_admin_subdomain
-    Apartment::Tenant.reset if request.subdomain == 'admin' || request.subdomain.blank?
+    Apartment::Tenant.reset if request.subdomain == 'admin' || !request.subdomain.present?
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.